junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -573,9 +573,11 @@ public static boolean isPackage(String packageName) {
 		Preconditions.notNull(clazz, "Class must not be null");
 		Preconditions.notNull(sortOrder, "MethodSortOrder must not be null");
 
-		// TODO [#333] Determine if we need to support bridged methods.
-
-		List<Method> localMethods = Arrays.asList(clazz.getDeclaredMethods());
+		// @formatter:off
+		List<Method> localMethods = Arrays.stream(clazz.getDeclaredMethods())
+				.filter(Method::isBridge) // [#333] don't collect bridge methods
```

Comment:
They did fail.

The initial commit contained an `@Disabled` test, a bunch commits later it still failed ... just the last (flawed) polishing and the last green lightbulb saying `BUILD SUCCESSFUL` convinced me that everything's alright. #648 will be done tonight.

Never stop learning.